addons/hr_skills_survey/models/survey_user.py

```diff
@@ -26,7 +26,7 @@ def _mark_done(self):
                 'employee_id': employee.id,
                 'name': survey.title,
                 'date_start': fields.Date.today(),
-                'date_end': fields.Date.today(),
+                'date_end': False,
```

Comment:
As your master changes are accompanied with a script that fixes these, can't we keep this change for the master version as well? Would make a bit more sense as a whole.

That way we have the minimal fix here.